agenda-services/src/main/java/org/exoplatform/agenda/listener/AgendaVotesListener.java

```diff
@@ -17,15 +24,27 @@
 
   private AgendaEventAttendeeService attendeeService;
 
+  private AgendaEventService         agendaEventService;
+
   public AgendaVotesListener(PortalContainer container) {
     this.container = container;
   }
 
   @Override
   public void onEvent(Event<Long, Long> event) throws Exception {
-    long eventId = event.getSource();
-    long identityId = event.getData();
-    getAttendeeService().sendEventResponse(eventId, identityId, EventAttendeeResponse.TENTATIVE);
+    ExoContainerContext.setCurrentContainer(container);
+    RequestLifeCycle.begin(container);
+    try {
+      long eventId = event.getSource();
+      long identityId = event.getData();
+      org.exoplatform.agenda.model.Event agendaEvent = getAgendaEventService().getEventById(eventId);
+      getAttendeeService().sendEventResponse(eventId, identityId, EventAttendeeResponse.TENTATIVE);
+      if (agendaEvent.getCreatorId() != identityId) {
```

Comment:
I would prefer to have a dedicated listener for VotesNotification. When the `event response` operation fails, the notification is send anyway.